### PR TITLE
Update manage-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by msbemba**
Global admins and Exchange admins can deploy the add-ins(requires org management role)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3908

File: microsoft-365/admin/manage/manage-deployment-of-add-ins.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Before you begin, see [Determine if Centralized Deployment of add-ins works for
   
 ### Considerations when assigning an add-in to users and groups
 
-Admins can assign an add-in to everyone or to specific users and groups. Each option has implications:
+Global Admins and Exchange Admins can assign an add-in to everyone or to specific users and groups. Each option has implications:
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-deployment-of-add-ins.md]:
```suggestion
Global admins and Exchange admins can assign an add-in to everyone or to specific users and groups. Each option has implications:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**